recipes/bolt/install_bolt.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -ex
+
+# temporary prefix to be able to install files more granularly
+mkdir temp_prefix
+
+if [[ "${PKG_NAME}" == "libbolt-devel" ]]; then
+    cmake --install ./build --prefix=./temp_prefix
+    # only bolt libraries
+    mkdir -p $PREFIX/lib/cmake/llvm
+    mv ./temp_prefix/lib/libLLVMBOLT* $PREFIX/lib
+    # only on linux-64
+    mv ./temp_prefix/lib/libbolt* $PREFIX/lib || true
+    # copy CMake metadata
+    mv ./temp_prefix/lib/cmake/llvm $PREFIX/lib/cmake/llvm
```

Comment:
Since `$PREFIX/lib/cmake/llvm` exists beforehand, you are moving this to `$PREFIX/lib/cmake/llvm/llvm`.
You want one of these:
```suggestion
    # copy CMake metadata
    cp ./temp_prefix/lib/cmake/llvm/. $PREFIX/lib/cmake/llvm/
```
```suggestion
    # move CMake metadata
    mv ./temp_prefix/lib/cmake/llvm/* $PREFIX/lib/cmake/llvm/
```
(Here the latter should suffice; you'd need something like the former with its `SOURCE/.` for files not covered by `*`, e.g. `.hidden`.)

Reply:
Thank you for reminding me about these arcane rules! 🙏

I actually had the latter first and then (mis-)remembered that I had gotten this wrong the last time.